_data/toc.yaml

```diff
@@ -1,5 +1,5 @@
 toc:
-- title: Welcome to the Docs
+- title: What is Docker?
```

Comment:
I prefer "Docker Documentation" or "Docker Docs". From a quick google I found "AWS Documentation", "Oracle Help Center", "Microsoft Technical Documentation", "Guides and Sample Code - Apple Developer",